rllib/utils/actor_manager.py

```diff
@@ -188,7 +191,9 @@ def __init__(
                 that cannot be scheduled because the limit has been reached will be
                 dropped. This only applies to the asynchronous remote call mode.
         """
-        self.__NEXT_ID = 0
+        # For historic reasons, just start remote worker ID from 1, so they never
+        # collide with local worker ID (0).
+        self.__NEXT_ID = 1
```

Comment:
sigh I don't like this. This has assumptions about where the implementation details of its caller (i.e WorkerSet) should we just add an input variable to constructor that tells what should be the init_id?

Reply:
oh, I like your idea a lot better! nice.